CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 Notable changes.
 
+## June 2023
+
+### [0.43.0]
+Fix a bug in passing users secrets to dotfile clone and install commands (https://github.com/devcontainers/cli/pull/544)
```

Comment:
Merge after #544 is in

Reply:
This commit will also be released if we release from main https://github.com/devcontainers/cli/commit/ae783be8ad0da7b51b7d5ead04b3b97a66888b55

Reply:
done